distutils/sysconfig.py

```diff
@@ -436,51 +436,6 @@ def expand_makefile_vars(s, vars):
 _config_vars = None
 
 
-_sysconfig_name_tmpl = '_sysconfigdata_{abi}_{platform}_{multiarch}'
```

Comment:
In #69, I introduced this property as a way for downstream packagers to customize (patch) the name of the sysconfig data. I probably at the same time should have added a comment to that effect or a unit test to protect this expectation.

Can you confirm that on Debian and pkgsrc distributions that the build_time_vars are honored?

Reply:
> Can you confirm that on Debian and pkgsrc distributions that the build_time_vars are honored?

I'm not sure what you mean by this. Can you explain a bit?

Reply:
The code that we're removing is replacing _get_config_vars() with the implementation from sysconfig. But at some point, when I tried supplying this version of distutils to older Pythons on NetBSD, I got a report (#16) that distutils' default scheme for resolving the `build_time_vars` (as in _init_posix) was no longer valid (that file didn't exist). Presumably that's not an issue, as `sysconfig.get_config_vars` should just work, but I want to be sure that it not only works, but that it isn't a downgrade from what distutils is currently providing.

Looking at that issue, I see mentions of the patches, and I can actually see how [sysconfig is patched](https://github.com/NetBSD/pkgsrc/blob/trunk/lang/python38/patches/patch-Lib_sysconfig.py) to honor the `_sysconfigdata_{sys.platform}` format.

So that checks out - this change is likely stable and limits the amount of patching/customization a downstream provider will need to supply.